### DISCUSSION TITLE
GitHub Code Search file size option

### DISCUSSION THREAD

**Discussion by YounesB-McGill** [community/community#22750] [Other Feature Feedback, Questions, & Ideas]
It is possible to specify a [file size](https://docs.github.com/en/search-github/searching-on-github/searching-code#search-by-file-size) using the regular search option on GitHub.com (eg, `size:>1000`). It would be great to add this option to the new Code Search experience on [cs.github.com](https://cs.github.com/)!

As of the time I write this, no such option is documented in the [syntax guide](https://cs.github.com/about/syntax), and trying to search by `size` gives the following warning:

> :warning: Possible unrecognized qualifier, searching for this term literally

**Comment by knackstedt** [comment]
A great example of why one may need to use this would be searching for .geojson files. Some can be 10+MB in size for more detailed or more comprehensive datasets, and that can't be filtered with just a path search.
